source/administration/compliance-export.rst

```diff
@@ -0,0 +1,43 @@
+Compliance Export Beta (E20 Add-On)
+=====================================
+
+Available as an `add-on to Enterprise Edition E20 <https://about.mattermost.com/pricing/>`_.
+
+This feature enables compliance exports to be produced from the System Console, with all query and download actions logged in an audit history to enable oversight and prevent unauthorized queries.
```

Comment:
What do you mean by "all query and download actions"? The reports contain all posts, including posts made in DM and GM channels, as well as file uploads, but they don't contain any information about search queries or who downloaded attachments

Reply:
I copied the sentence from the Compliance doc and meant to update it for our export doc. I'll do that, good catch